app/config/FrontendAppConfig.scala

```diff
@@ -82,6 +82,18 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
 
   lazy val flatRateExpenseId: Int = configuration.get[Int]("flatRateExpenseId")
 
+  lazy val accessibilityStatementUrl: String = configuration.get[String]("accessibilityStatement.govAccessibilityStatementUrl")
+  lazy val abilityNettUrl: String = configuration.get[String]("accessibilityStatement.abilityNetUrl")
+  lazy val w3StandardsUrl: String = configuration.get[String]("accessibilityStatement.w3StandardsUrl")
+  lazy val equalityAdvisoryServiceUrl: String = configuration.get[String]("accessibilityStatement.equalityAdvisoryServiceUrl")
+  lazy val equalityNIUrl: String = configuration.get[String]("accessibilityStatement.equalityNIUrl")
+  lazy val dealingHmrcAdditionalNeedsUrl: String = configuration.get[String]("accessibilityStatement.dealingHmrcAdditionalNeedsUrl")
+  lazy val dacUrl: String = configuration.get[String]("accessibilityStatement.dacUrl")
+
+  val accessibilityStatementLastTested: String = configuration.get[String]("accessibilityStatement.lastTested")
+  val accessibilityStatementFirstPublished: String = configuration.get[String]("accessibilityStatement.firstPublished")
```

Comment:
The other Val's are lazy, is there any difference between these configs over the others?

Reply:
not a major thing, but if the urls aren't in config we would want to stop compiling rather than wait for someone to open a page and have it break